changelogs/fragments/bugfix_178.yaml

```diff
@@ -0,0 +1,3 @@
+---
+doc_changes:
```

Comment:
This is a trivial non-user facing change. You can remove the changelog fragment and add `skip-changelog` label.